db/schema.rb

```diff
@@ -105,14 +105,14 @@
   add_index "order_lines", ["product_id"], name: "index_order_lines_on_product_id", using: :btree
 
   create_table "orders", force: :cascade do |t|
-    t.integer  "user_id",    null: false
-    t.integer  "group_id",   null: false
+    t.integer  "user_id",        null: false
+    t.integer  "group_id",       null: false
+    t.datetime "confirm_before", null: false
     t.datetime "created_at"
     t.datetime "updated_at"
   end
 
-  add_index "orders", ["group_id"], name: "index_orders_on_group_id", using: :btree
-  add_index "orders", ["user_id"], name: "index_orders_on_user_id", using: :btree
+  add_index "orders", ["user_id", "group_id", "confirm_before"], name: "index_orders_on_user_id_and_group_id_and_confirm_before", using: :btree
```

Comment:
I don't see these there attributes used in any query. If so, I'd add only the id columns in the index